docker-storage-setup.1

```diff
@@ -31,14 +31,11 @@ functionality to work properly.
 \f[B]Supported options for the configuration file\f[]:
 
 STORAGE_DRIVER:
-      Specify a storage driver to be used with docker. Default
-      driver is "devicemapper". Other valid values are overlay,
-      overlay2 and "". Empty string tells docker-storage-setup to
-      not do any storage setup.
+      Specify a storage driver to be used with container runtime. The default driver is "devicemapper". Valid values are overlay, overlay2 and "". Empty string tells docker-storage-setup to not perform any storage setup.
```

Comment:
Do you like this new format. I mean when I do man, start of description line is aligned with the option this text is explaining.

I liked previous format better where all text was shifted to right.